src/lib/logger/impl.ml

```diff
@@ -2,7 +2,7 @@ open Core
 open Async
 
 module Level = struct
-  type t = Trace | Debug | Info | Warn | Error | Faulty_peer | Fatal
+  type t = Trace | Debug | Info | Warn | Error | Faulty_peer | Fatal | Spam
```

Comment:
This type definition is enumerated, so order matters. Spam seems like it should be at the beginning here because is is less important and noisier than all the other levels.